srcpkgs/firebird3/template

```diff
@@ -89,7 +89,7 @@ pre_configure() {
 	# do not change ownership to root:root and
 	# do not change permissions to read-only
 	find -iname "*.sh.in" -exec sed -i "{}" \
-		-e 's;"`whoami`";"root";' \
+		-e 's;"$(whoami)";"root";' \
```

Comment:
Nope, the backtick is correct, we're trying to replace the execution with `root`, you may want to write a patch instead.